index.html

```diff
@@ -40,6 +41,185 @@ <h1>Hey there. I'm Sainingo <br><span>I'm a software developer</span></h1>
               </ul>
 
           </section>
-        </main>
+     </main>
+
+ 
+     <section class="projects">
+        <div class="project-flex">
+            <span class="disable-icon"><img src="./assets/images/Disabled.png" alt="disable" ></span>
+            <h2>My Recent Works</h2>
+            <span><img src="./assets/images/Indicator.png" alt="indicator"></span>
+        </div>
```

Comment:
- [ ] [OPTIONAL] Just above "My Recent Works", there is a big white space that should not be in the design. Kindly remove it.
![Screenshot from 2022-02-16 10-48-54](https://user-images.githubusercontent.com/58692590/154239985-949db46c-e806-453a-9f41-bea0f8bc2aac.png)

Reply:
Ok. Thanks for the feedback